### ISSUE TITLE
Climsoft Import Wizard Dialog Checkboxes Labels

### ISSUE THREAD

**Issue by Patowhiz** [africanmathsinitiative/R-Instat#6986]
The set label text for checkboxes in the climsoft import wizard dialog are not updated.

The checkboxes should show the labels set in the code.

**Comment by rdstern** [comment]
@Patowhiz I assume this can be replaced by deleting the menu item for the wizard?
@N-thony here is the current menu.

![image](https://github.com/user-attachments/assets/eb84f406-ad68-4661-8770-53678e6076cf)

I suggest simply:
a) Delete the Wizard item
b) Delete the line above Import from Climsoft.

(Then Climsoft is the last importing dialog and the first exporting one.  I think that's ok.)

@N-thony please can you allocate.

**Comment by Patowhiz** [comment]
@rdstern yes. And also deleting the wizard dialog and it's underlying R code. Let me quickly do this while I fix the bugs that we found when using station names. 
I found the bug is raised when you have station names that have apostrophes. SQL needs them to be escape.
